src/components/myNotes/actions/filters/FilterDatesForm.jsx

```diff
@@ -0,0 +1,47 @@
+import {
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  FormGroup,
+  FormControl,
+  FormControlLabel,
+  Button,
+  Checkbox,
+} from "@mui/material";
+import PropTypes from "prop-types";
+
+import styles from "pages/styled";
+
+const FilterDatesForm = ({ filterDateOpen, setFilterDateOpen, uniqueDates, changeDateFilters, applyDateFilters }) => (
+  <Dialog open={filterDateOpen} onClose={() => setFilterDateOpen(false)}>
```

Comment:
Use please previous values inside function of changing state
```suggestion
  <Dialog open={filterDateOpen} onClose={changeFilterDateOpen}>
```